pypeit/core/telluric.py

```diff
@@ -705,8 +713,11 @@ def eval_sensfunc_model(theta, obj_dict):
 
     Returns
     -------
-    counts_model : array with same shape as obj_dict['wave_star']
-       Model of the quantity being fit for the sensitivity function which is exptime*f_lam_true/sensfunc
+    counts_per_angstrom_model : array with same shape as obj_dict['wave_star']
```

Comment:
Make sure `fluxing.rst` is updated accordingly.  
And also tickle the related Issue.

Reply:
Updated fluxing.rst and resolving the issue.